irc/websocket.go

```diff
@@ -0,0 +1,56 @@
+package irc
+
+import (
+	"github.com/gorilla/websocket"
+	"net"
+	"net/http"
+	"time"
+)
+
+var upgrader = websocket.Upgrader{
+	ReadBufferSize:  1024,
+	WriteBufferSize: 1024,
+	CheckOrigin:     func(r *http.Request) bool { return true },
+}
+
+type WSContainer struct {
```

Comment:
Would it make more sense to call this type `WSConn` to be consistent with the interface it's implementing, http://golang.org/pkg/net/#Conn ?